_posts/2020-12-01-infrastructure-migration-finished.md

```diff
@@ -0,0 +1,318 @@
+---
+title: Infrastructure migration finished
+author: TrueBrain
+---
+
+Today is a good day.
+Today is the day all openttd.org services run on [AWS](https://aws.amazon.com/).
+Today is the day I can finally shut down our [OVH dedicated server](https://www.ovhcloud.com/en-gb/bare-metal/).
+Today is the day I can finally retire software like:
+- [Django](https://www.djangoproject.com/) 1.2 (released May 2010)
+- [Debian](https://www.debian.org/) Etch (release April 2007)
+- [MediaWiki](https://www.mediawiki.org/) 1.12.0 (released winter 2008)
+- [XenServer](https://xenserver.org/) 6.5 (released January 2015)
+- An IRC bot that was running since 2016.
```

Comment:
IRC bot since 2016? Do you mean 2006? I would have guessed 2007.

Reply:
It was **running** since 2016, non-stop. I will reword :)